website/content/docs/k8s/multiport/reference/trafficpermissions.mdx

```diff
@@ -0,0 +1,334 @@
+---
+layout: docs
+page_title: TrafficPermissions resource configuration reference
+description: The TrafficPermissions CRD defines rules for allowing and denying traffic between services within the service mesh. Learn how to configure traffic permissions for the v2 catalog to authorize service-to-service communication in a network with zero-trust security.
+---
+
+# TrafficPermissions configuration reference
+
+This page provides reference information for the TrafficPermissions resource, which authorizes east/west traffic between services within the service mesh. The traffic permissions CRD replaces the service intentions CRD when using the v2 catalog API. Refer to [changes to Consul's existing architecture](/consul/docs/architecture/catalog/v2#changes-to-consul-s-existing-architecture) for more information.
+
+This custom resource definition (CRD) describes a GAMMA resource that requires the [v2 catalog API](/consul/docs/architecture/catalog/v2). It is not compatible with the [v1 catalog API](/consul/docs/architecture/catalog/v1). For more information about GAMMA resources, refer to the [Kubernetes Gateway API documentation](https://gateway-api.sigs.k8s.io/concepts/gamma/).
+
+## Configuration model
+
+The following list outlines field hierarchy, language-specific data types, and requirements in a TrafficPermissions CRD. Click on a property name to view additional details, including default values.
+
+<Tabs>
+
+<Tab heading="YAML" group="yaml">
+
+- [`apiVersion`](#apiversion): string | required | must be set to `auth.consul.hashicorp.com/v2beta1`
+- [`kind`](#kind): string | required | must be set to `TrafficPermissions`
+- [`metadata`](#metadata): object | required
+  - [`name`](#metadata-name): string | required
+  - [`namespace`](#metadata-namespace): string | optional <EnterpriseAlert inline />
+- [`spec`](#spec): object | required
+  - [`destination`](#spec-destination):
+    - [`identityName`](#spec-destination-identityname):
+  - [`action`](#spec-action): string
+  - [`permissions`](#spec-permissions): list of maps
+    - [`sources`](#spec-permissions-sources):
+      - [`identityName`](#spec-permissions-sources-identityname): string
+      - [`namespace`](#spec-permissions-sources-namespace): string
+      - [`partition`](#spec-permissions-sources-partition): string
+      - [`peer`](#spec-permissions-sources-peer): string
+      - [`samenessGroup`](#spec-permissions-sources-samenessgroup): string
+      - [`exclude`](#spec-permissions-sources-exclude): Map
+        - [`identityName`](#spec-permissions-sources-exclude): string
+        - [`namespace`](#spec-permissions-sources-exclude): string
+        - [`partition`](#spec-permissions-sources-exclude): string
+        - [`peer`](#spec-permissions-sources-exclude): string
+        - [`samenessGroup`](#spec-permissions-sources-exclude): string
+    - [`destinationRules`](#spec-permissions-destinationrules):
```

Comment:
Within destinationRules, only portNames are supported at the moment since the rest of the properties are related to L7, I think this means we should include something here to signal that these are unsupported fields. [RFC link](https://docs.google.com/document/d/1XWZoGwMYV0VDZlssiCRRkvea7r6tD2VZZoaDOYjBudc/edit#heading=h.9t84kt6vey7k)